image/templates/helm/stackrox-secured-cluster/templates/service-ca.yaml

```
@@ -13,4 +13,7 @@ metadata:
 type: Opaque
 stringData:
   ca.pem: |
+    {{- if hasKey (.Values.metadata).annotations "init-bundle.stackrox.io/name" }}
```

Are the parentheses needed?
```suggestion
    {{- if hasKey .Values.metadata.annotations "init-bundle.stackrox.io/name" }}
```

---

Hey, thank you for your comments, I have a few clarifications/questions to make:

I utilized the  {{- if hasKey (.Values.metadata).annotations "init-bundle.stackrox.io/name" }} because when you cat the helm and operator bundle yaml files they look almost identical except for the indicator that the operator bundle contains "init-bundle.stackrox.io/name" difference. Which when I ran the helm template command for both bundle formats I got the desired output  with the correct error outputs. Could you further explain with what you mean by using the  ._rox.ca._cert in this case?

And as for the (), originally I had it as  {{- if hasKey .Values.metadata.annotations "init-bundle.stackrox.io/name" }} however when I ran the test case there was an error compiling it because of syntax factors and the () was needed. So with the (), it compiles correctly.

And I will add the manual test again, before the PR i had tried it with both operator and init bundles, got the expected error for the operator bundle, and Ill include the commands I used/with the output for both cases again. 

Thank you, And let me know about the information above,
Amina Kameric

---

@porridge I think that without parenthesis, there was a nil pointer exception (because `.Values.metadata` might be nil)